desktop/extensions-sdk/build/minimal-frontend-extension.md

```diff
@@ -65,18 +66,44 @@ A `metadata.json` file is required at the root of the image filesystem.
 {
   "ui": {
     "dashboard-tab": {
-      "title": "Min FrontEnd Extension",
+      "title": "Minimal frontend",
```

Comment:
This new title doesn't match the screenshot, nor the [sample file](https://github.com/docker/extensions-sdk/blob/main/samples/minimal-frontend/metadata.json)

<img width="258" alt="image" src="https://user-images.githubusercontent.com/35727626/208386760-c1ce4ff1-0e28-41e3-8a86-9826dfa21a23.png">

Reply:
Good catch! I'll update the screenshot

Reply:
Done